docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -170,78 +149,197 @@ correct Google Workspace plan.
-    ![OAuth client data](../../../img/googleoidc/clientid-data.png)
+![OAuth client ID creation](../../../img/sso/googleoidc/clientid-creation.png)
+
+Copy the Client ID and Client Secret from the next screen or click **DOWNLOAD JSON**:
+
+![OAuth client data](../../../img/sso/googleoidc/clientid-data.png)
 
 ### Create a service account
 
-On the [Create a service account](https://console.cloud.google.com/iam-admin/serviceaccounts/create) page, pick a name for your service account. Leave project access grants and user access grants empty.
-    ![service account creation](../../../img/googleoidc/serviceacct-creation.png)
+From the GCP **IAM & Admin** menu select **Service Accounts**. From the kebab
+menu click **CREATE SERVICE ACCOUNT**:
+
+![Create a service account](../../../img/sso/googleoidc/create-service-account.png)
+
+On the [Create a
+service account](https://console.cloud.google.com/iam-admin/serviceaccounts/create)
+page, pick a name for your service account. Leave project access grants and user
+access grants empty.
 
-  Click the newly-created account to view its details, and copy the Unique ID for later.
-    ![service account unique ID](../../../img/googleoidc/serviceacct-uniqueid.png)
+![service account creation](../../../img/sso/googleoidc/serviceacct-creation.png)
 
-  Create a new key for the service account, select JSON as the key type, and save the resulting JSON file.
-    ![service account key creation](../../../img/googleoidc/serviceacct-key.png)
-    Later, we will make this JSON available to the Teleport Auth Server via the OIDC Connector configuration, either by referencing a local file or pasting the JSON into the configuration YAML. If you plan to take the first approach, you will need to upload the JSON to the Auth Server.
+Click the newly-created account to view its details, and copy the Unique ID for later.
+
+![service account unique ID](../../../img/sso/googleoidc/serviceacct-uniqueid.png)
+
+Create a new key for the service account, select JSON as the key type, and save
+the resulting JSON file.
+
+![service account key creation](../../../img/sso/googleoidc/serviceacct-key.png)
+
+Later, we will make this JSON available to the Teleport Auth Server via the
+OIDC Connector configuration, either by referencing a local file or pasting the
+JSON into the configuration YAML. If you plan to take the first approach, you
+will need to upload the JSON to the Auth Server.
 
 <Admonition type="note">
-  Teleport requires the service account JSON to be available to all Teleport Auth Server hosts when deploying Teleport in a High Availability configuration. Unless you paste the JSON into the OIDC Connector configuration, you will need to upload the JSON file to all Auth Server hosts.
+Teleport requires the service account JSON to be available to all Teleport Auth
+Server hosts when deploying Teleport in a High Availability configuration.
+Unless you paste the JSON into the OIDC Connector configuration, you will need
```

Comment:
We may want to advise `--google-acc` flag here, instead of pasting JSON, possibly using the syntax `"$(< file.json)"`
(see: https://stackoverflow.com/a/4229151).

Reply:
Updated to expand the explanation of the differences and make embedded JSON the preference: 83ddab8d00a9cbf02ddd57722c05339e0c8fe373